libs/ipc/inc/public/tfc/ipc/details/impl.hpp

```diff
@@ -209,8 +210,15 @@ class slot : public transmission_base<type_desc> {
   auto connect(std::string_view signal_name) -> std::error_code {
     // TODO: Find out if these mutexes inside optimize single threaded are really needed
     socket_ = azmq::sub_socket(socket_.get_io_context(), true);
-    boost::system::error_code error_code;
     std::string const socket_path{ utils::socket::zmq::ipc_endpoint_str(signal_name) };
+
+    boost::system::error_code error_code;
+    if (socket_.set_option(
+            azmq::socket::reconnect_ivl(std::chrono::duration_cast<std::chrono::milliseconds>(reconnect_interval).count()),
```

Comment:
Remove duration cast according to other comment

Reply:
I would like to keep the duration_cast even if the type happens to match. It is clear of intent. And documents the resolution of ivl_reconnect for free.